client/package.json

```
@@ -8,8 +8,9 @@
     "build": "vite build",
     "lint": "eslint .",
     "preview": "vite preview",
-    "test": "vitest run",
-    "coverage": "vitest run --coverage"
+    "test": "vitest",
```

Keep this line as before.
```suggestion
    "test": "vitest run",
```